lib/TorchConversion/Transforms/VerifyTcpBackendContract.cpp

```diff
@@ -7,10 +7,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "PassDetail.h"
+#include "../PassDetail.h"
 
-#include "IR/TcpDialect.h"
-#include "IR/TcpOps.h"
+#include "Dialect/IR/TcpDialect.h"
+#include "Dialect/IR/TcpOps.h"
```

Comment:
These are not being tested at the moment, I plan to send a follow-on to add `TorchBackendToTcpBackendPipeline` along with LIT tests that should cover this pass.